apps/search/index.html

```diff
@@ -95,9 +86,17 @@
         <gaia-grid cols="4" text-rows="3" id="icons"></gaia-grid>
       </section>
       <div id="loading"><progress></progress></div>
+
       <h2 id="places-header" class="hidden"><span data-l10n-id="history"></span></h2>
       <section id="places" class="local" data-l10n-id="places"
                aria-label="Places"></section>
+
+      <div id="suggestions-wrapper">
```

Comment:
What's this wrapper used for? It's a bit inconsistent with the Places section which doesn't have a wrapper.